application/scss/components/_subnav.scss

```diff
@@ -21,17 +21,18 @@
   }
 }
 
-.app-subnav-toggle__label {
+.app-subnav-toggle__button {
```

Comment:
Was it possible to add make the button hidden if there's no JS (without the no-js class like GOV.UK have) and then display: inline-block with `.js-enabled .app-subnav-toggle__button`?